CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ option(INEXOR_BUILD_BENCHMARKS "Build benchmarks" OFF)
 option(INEXOR_BUILD_DOC "Build documentation" OFF)
 option(INEXOR_BUILD_EXAMPLE "Build example" ON)
 option(INEXOR_BUILD_TESTS "Build tests" OFF)
+option(INEXOR_BUILD_WITH_CHECKS "Build tests" OFF)
```

Comment:
Please add the value of this option below to the messages.
```
message(STATUS "INEXOR_BUILD_WITH_CHECKS = ${INEXOR_BUILD_WITH_CHECKS}")
```
Would be nice if you add the missing space to `INEXOR_BUILD_TESTS` also ^^